index.rst

```diff
@@ -191,6 +195,22 @@ Since use of "extent" is necessary for displaying a WCS overlay for ``deepCoadds
   plt.imshow(deepCoadd.image.array, cmap='gray', vmin=0, vmax=2, extent=deepCoadd_extent, origin='lower')
 
 
+Functions and classes
+---------------------
+
+While globally defined functions or classes which are used more than once in a notebook should be
+defined in section ``1.2 Define Functions and Parameters``, single-use functions or classes 
+can be defined immediately before they are used.
+
+Functions or classes that are particularly long blocks of code (e.g., >20 lines) can be hidden by going to
+the "View" menu item and choosing "Collapse Selected Code", or by clicking on the blue bar that
+appears to the left of a selected cell.
+Hidden cells should be described in the preceding markdown cell with text like 
+"the following hidden cell contains code that defines the ``make_cmd_plot`` function".
+The first hidden cell in a notebook should include instructions for displaying the cell, such as
+"to see the contents of the hidden cell, select View from the menu bar and then Expand Selected Code".
```

Comment:
We could also explain that there are direct ways of expanding the cell with something like: "To see the contents of the hidden cell, select View from the menu bar and then Expand Selected Code, or click on the vertical next to the cell or on the three dots that denote that the cell is hidden."

Reply:
implemented verbatim, thank you!